include/nanoflann.hpp

```diff
@@ -1801,6 +1801,10 @@ namespace nanoflann
 			index=index_;
 		}
 
+	private:
+		/** Hidden copy constructor, to disallow copying indices (Not implemented) */
+		KDTreeSingleIndexDynamicAdaptor(const KDTreeSingleIndexDynamicAdaptor<Distance, DatasetAdaptor, DIM, IndexType>&);
```

Comment:
Since we support C++11, I think it would be clearer to make this one public with an `= delete;` keyword.